charts/metering-ansible-operator/templates/crds/meteringconfig.crd.yaml

```diff
@@ -94,23 +94,40 @@ spec:
                       - container
                       - secretName
                       properties:
+                        createSecret:
+                          type: boolean
                         container:
                           type: string
+                          minLength: 1
                         secretName:
                           type: string
+                          minLength: 1
+                        storageAccountName:
+                          type: string
+                          minLength: 1
+                        secretAccessKey:
+                          type: string
+                          minLength: 1
                         rootDirectory:
                           type: string
+                          minLength: 1
                     sharedPVC:
                       type: object
                       properties:
+                        createPVC:
+                          type: boolean
                         claimName:
                           type: string
+                          minLength: 1
                         storageClass:
                           type: string
+                          minLength: 1
                         size:
                           type: string
-                        createPVC:
```

Comment:
just moving this property to match the convention of other keys throughout the CRD.